api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -29,19 +31,24 @@
 public class FireCloudServiceImpl implements FireCloudService {
   private static final Logger log = Logger.getLogger(FireCloudServiceImpl.class.getName());
 
+  private static final String REGISTERED_GROUP_NAME = "ALL-OF-US-REGISTERED";
```

Comment:
We may want to put this in WorkbenchConfig somewhere, so it can vary by environment.

I think we want this to be lowercase, too.